presto-session-property-managers/src/main/java/io/prestosql/plugin/session/db/DbSessionPropertyManagerConfig.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.session.db;
+
+import io.airlift.configuration.Config;
+import io.airlift.units.Duration;
+import io.airlift.units.MinDuration;
+
+import static java.util.concurrent.TimeUnit.SECONDS;
+
+public class DbSessionPropertyManagerConfig
+{
+    private String configUrl = "jdbc:mysql://localhost:3306";
+    private String username = "user";
+    private String password = "password";
+    private Duration specsRefreshPeriod = new Duration(5, SECONDS);
+
+    public String getConfigDbUrl()
```

Comment:
Add `@NotNull` for all of these. This, combined with the null default values, will enforce that they are configured.